### PR TITLE
Public read me and developer read me 

### PR DESCRIPTION
**Pull request by Solit95**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +93,19 @@ To restore Models to a prior version, execute the following CLI command:
 
 > `denogres --restore [PREVIOUS_MODEL_FOLDER]`
 
+## Testing
+
+To run all tests, use the command:
```
Comment (rcheman) [inline README.md:98]: Does a user of DenoGres need to know how to run the tests? They shouldn't be touching anything that breaks its functionality and would need to be tested.

File: deno.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
     "appDEV": "deno run -Ar --unstable \"https://deno.land/x/denogresdev/webview/webview.ts\"",
     "appPROD": "deno run -Ar --unstable \"https://deno.land/x/denogres/webview/webview.ts\"",
     "test": "deno test --no-check --allow-read --allow-net --allow-env --allow-write --allow-run --no-remote",
+    "model-tests": "deno test -A --no-check ./Test/model_test.ts && ./Test/model_integration_tests/model_integration_test.ts",
     "playground": "deno run -A ./playground/playground.ts"
```
Comment (rcheman) [inline deno.json:8]: Do we want to still include the playground script or is this to be left in until right before launch?

File: developerReadMe.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+to continue this work or deploy Denogres 3.0, in which case you'll find a more
+detailed readme in the denogres-gui directory in this respository.
+
+## <u>Getting Started</u>
+
+Most of the code for the ORM can be found in the root directory and src
+directory. In the root directory, you'll find a `mod.ts` file that defines the
+behavior of the Denogres CLI (including denogres --init, --db-pull, etc.), as
+well as a `deps.ts` file where, following Deno best practices, we've imported
+and exported all dependencies to for the rest of the application.
```
Comment (alexanderlin) [inline developerReadMe.md:27]: should to be too?

Comment (mjpg1) [reply developerReadMe.md]: to should be an empty string, it should not exist (J's changing it now) -- also the Vendor section is waiting for you!

File: developerReadMe.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+- Theoretically, it would be interesting to incorparate compatibility with other
+  popular database management systems such as MySQL, but based on the structure
+  of the mdoel class (which uses [Deno Postgres](https://deno-postgres.com/#/)
+  under the hood), this would be a large undertaking.
+
+## <u>Deployment</u>
+
+The Denogres-2.0 GUI and documentation website/landing page are already deployed
+through Deno Deploy. Any changes made to the main branch of the DenoGres/Website
+repo (linked above) are automatically deployed.
```
Comment (rcheman) [inline developerReadMe.md:97]: This whole developer ReadMe is just excellent! I really appreciate all the intentionality shown here and I think it leaves future developers in a really good spot to hit the ground rolling.

File: README.md (hunk 1 above)
Comment (mjpg1) [reply README.md:98]: I believe the README is for developers who want to contribute to the open source project (in addition to obviously users of the developer tool)--something James mentioned today!

File: deno.json (hunk 2 above)
Comment (mjpg1) [reply deno.json:8]: Won't be included in the final version!  (already in Asana board)

File: developerReadMe.md (hunk 4 above)
Comment (mjpg1) [reply developerReadMe.md:97]: <3

File: README.md (hunk 1 above)
Comment (rcheman) [reply README.md:98]: Gotcha, I guess I misunderstood that and thought that was why you had separate a developer readMe and regular readMe.

File: deno.json (hunk 2 above)
Comment (rcheman) [reply deno.json:8]: Just double checking!

**Review by rcheman — APPROVED** [review]

**Pull request merged**